### PR TITLE
[DOCS]chore: refactor sidemenu to vanilla extract

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  sprinkles({
+    paddingTop: '$4',
+    paddingBottom: '$2',
+  }),
+  {
+    borderBottom: `1px solid ${vars.colors.$borderDefault}`,
+  },
+]);
+
+export const sidemenuTitleClass = style([
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts:33]:
```suggestion
export const sideMenuTitleClass = style([
```

File: packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    paddingLeft: '$8',
+    textAlign: 'left',
+  }),
+  {
+    background: 'transparent',
+    border: '0',
+  },
+]);
+
+export const sidemenuTitleButtonClass = style([
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts:46]:
```suggestion
export const sideMenuTitleButtonClass = style([
```

File: packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        borderLeft: `2px solid ${vars.colors.$borderDefault}`,
+        borderBottom: `2px solid ${vars.colors.$borderDefault}`,
+        transform: `translate(${vars.sizes.$2}, ${vars.sizes.$2}) rotate(45deg)`,
+        transition: 'transform .2s ease ',
+      },
+    },
+  },
+]);
+
+export const linkClass = style([sprinkles({}), {}]);
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts:80]: why do you need this empty class? 

nitpick: 
even if we need empty class you can set it up without `sprinkles`

`export const linkClass = style({});`

or you can also use 

`export const linkClass = 'side-menu-link-class'`

File: packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import { sprinkles, vars } from '@kadena/react-ui/theme';
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts:1]: nitpick: file name could be `sideMenu.css.ts`

File: packages/apps/docs/src/components/Layout/components/SideMenu/menucard.css.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { sprinkles } from '@kadena/react-ui/theme';
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/SideMenu/menucard.css.ts:1]: nitpick: 
file name could be `menuCard.css.ts`

File: packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts (hunk 3 above)
Comment (sstraatemans) [reply packages/apps/docs/src/components/Layout/components/SideMenu/sidemenu.css.ts]: should not be there..sorry

**Review by realdreamer — APPROVED** [review]

**Pull request merged**